x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -16,7 +16,8 @@ export const policyFactory = (
   cloud = false,
   licenseUid = '',
   clusterUuid = '',
-  clusterName = ''
+  clusterName = '',
+  serverless = false
```

Comment:
consider refactoring arguments into an object. The list is getting too long, isn't it? :P

Reply:
definitely agree. I wanted to do this but saw that it involved 30+ files so didn't want to do it in this PR 😅.